images/linux/scripts/installers/pulumi.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  pulumi.sh
+##  Desc:  Installs Pulumi
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install Pulumi
+curl -fsSL https://get.pulumi.com | sh
+cp -r ~/.pulumi/bin/* /usr/local/bin/
```

Comment:
Currently, `$HOME/.pulumi` has the following structure:
```
$HOME/.pulumi
---- bin
-------- pulumi
-------- pulumi-analyzer-policy
-------- pulumi-analyzer-policy-python
-------- pulumi-language-dotnet
-------- pulumi-language-go
-------- pulumi-language-nodejs
-------- pulumi-language-python
-------- pulumi-language-python-exec
-------- pulumi-resource-pulumi-nodejs
-------- pulumi-resource-pulumi-python
```
And this code work as expected for now.
But I wonder why `$HOME/.pulumi` is created by pulumi installer:
- can it be used somehow during pulumi run?
- can something be changed in future and this folder will contain additional folders to `bin`

cc: @al-cheb @miketimofeev

Reply:
Hi, Maxim!
Pulumi documentation for manual installation on Linux
https://www.pulumi.com/docs/get-started/install/
tells: “Extract the tarball and move the binaries in the pulumi directory to a directory included in your system’s $PATH.”. 
So I guess that copy binaries to /usr/local/bin (that path is included in $PATH for all users) is correct in this case. 
Anyway, I’ll be happy to change anything if I’m wrong, thank you!

Reply:
In that case, maybe it makes sense to use manual installation?
https://www.pulumi.com/docs/get-started/install/#manual-installation
Just download the latest tar and unpack it to `/usr/local/bin`?

Reply:
@LeonidLapshin, thank you for explanation. In this case, your code looks good to me.
But I think I agree with @miketimofeev suggestion

Reply:
@miketimofeev, thank you for your suggestion, I made some changes within script, the installation logic is no more hidden behind a downloadable file. 
@maxim-lobanov, I hope, it looks clearer than before.